cluster/engine_test.go

```diff
@@ -416,8 +416,8 @@ func TestUsedCpus(t *testing.T) {
 
 	for _, hn := range hostNcpu {
 		for _, cn := range containerNcpu {
-			if int(cn) <= hn {
-				mockInfo.NCPU = hn
+			if cn <= hn {
+				mockInfo.NCPU = int(hn)
```

Comment:
I am afraid no need to `mockInfo.NCPU = int(hn)`, hn already has a type of `int64`.
In L438, code converts it back to `int64`. Right?